guides/common/modules/proc_salt-running-salt.adoc

```diff
@@ -0,0 +1,50 @@
+[id="salt_guide_running_salt_{context}"]
+= Running Salt
+
+You can run arbitrary Salt functions using {Project}'s remote execution features.
+
+Most commonly, you can run `salt.highstate` on one or more Salt Minions.
+This applies all relevant Salt States on your managed hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts*.
+. Select one or multiple hosts.
+. Click *Run Salt* from the _actions_ drop down menu.
++
+Alternatively, you can click *Run Salt* from the _Schedule Remote Job_ drop down menu on the host overview page.
+
+Triggering a remote execution job takes you to the corresponding job overview page.
+
+If you select a host from the list, you can see the output of the remote job in real time.
+
+To view remote jobs, in the {ProjectWebUI}, navigate to *Monitor > Jobs*.
+
+Running `state.highstate` generates a {ManagingHostsDocURL}salt_guide_salt_reports[Salt report].
+Note that reports are uploaded to {Project} every ten minutes.
+
+You can use {Project}'s remote execution features to run arbitrary Salt functions.
+This includes an optional test mode that tells you what would happen without actually changing anything.
+
+.Procedure
+. Navigate to *Hosts > All Hosts*
+. Select a host.
+. Click *Schedule Remote Job*.
+. For the *Job category* field, select _Salt_ from the drop down menu.
+. For the *Job template* field, select _Salt Run function_ from the drop down menu.
+. Fill in the function you want to trigger, for example `pillar.items` or `test.ping`.
```

Comment:
```suggestion
. Complete the function you want to trigger, for example `pillar.items` or `test.ping`.
```
Normally, you'd mention the name of the field name. I see this applies to both, but is there any label on the text field?

Reply:
The field is called `function`. Should I keep the capitalization?

Reply:
Obvs non-blocking but the way I would write it is:
"In the *Function* field, enter the name of the function that you want to trigger, for example, `pillar.items` or `test.ping`."

Just I used that Graphical User Interface Elements chapter of IBM style as a guide for so long. But this is fine.

Reply:
Function should be bold not italics in my example ^

Reply:
Thanks, applied.